### PR TITLE
Im an idiot

### PR DESCRIPTION
**Pull request by JLinkous15**
Closes Ticket #63 

# Adds functionality to Delete Button and shamed developer for working in main

---

# Steps to Test

1. Fetch the Im-an-idiot branch and switch to it
2. Navigate to Posts in browser
3. Navigate to Post Details by clicking on post title link
4. Click Delete Button
5. Eat Cake

### CODE REVIEW

**Comment by jesmeeker** [comment]
I can only delete from all posts, but functionality is not on the myPosts page. I checked and the ticket specifies this functionality on the myPosts page. Should it also be on the post details page (where it is now)?

File: src/components/posts/PostDetail.js [hunk 1]
```diff
@@ -26,7 +27,11 @@ export const PostDetail = ({token}) => {
         <div className="buttons">
             <button
             onClick={
-                ()=>{}
+                (e)=>{
+                    e.preventDefault()
+                    deletePosts(postId)
```
Comment (jesmeeker) [inline src/components/posts/PostDetail.js:32]: I think this feature should be in the myposts module. Not sure it belongs here in the post details.

**Review by jesmeeker — CHANGES_REQUESTED** [review]

**Comment by JLinkous15** [comment]
Thank you. Ready for review.

**Review by jesmeeker — APPROVED** [review]

**Review by jesmeeker — CHANGES_REQUESTED** [review]

**Review by jesmeeker — APPROVED** [review]

**Comment by jesmeeker** [comment]
You're good to go!

**Pull request merged**